openwisp_monitoring/device/tests/__init__.py

```diff
@@ -95,8 +91,8 @@ def _create_multiple_measurements(self, create=True, no_resources=False, count=4
         data3 = deepcopy(data)
         data3['interfaces'][0]['statistics']['rx_bytes'] = 500000000
         data3['interfaces'][0]['statistics']['tx_bytes'] = 300000000
-        data3['interfaces'][1]['statistics']['rx_bytes'] = 0
-        data3['interfaces'][1]['statistics']['tx_bytes'] = 0
+        data3['interfaces'][1]['statistics']['rx_bytes'] = 1
+        data3['interfaces'][1]['statistics']['tx_bytes'] = 1
```

Comment:
Why did these values change?

Reply:
I had added a comment above it stating that I am working on it. I wasn't able to pinpoint the issue then. The problem was with the views, fixed it and added a comment for the same :smiley:.